src/main/java/duke/DateTimeHandler.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Encapsulates a class that handles parsing of date-time inputs
+ */
+public class DateTimeHandler {
+    String[] formats = {
+            "dd-MM-yyyy hh:mm a",
+            "dd-MM-yyyy HHmm",
+            "dd/MM/yyyy hh:mm a",
+            "dd/MM/yyyy HHmm"
+    };
+    private final DateTimeFormatter[] dtfList = new DateTimeFormatter[formats.length];
+
+    public DateTimeHandler() {
+        for(int i=0; i<formats.length; i++) {
```

Comment:
Could possibly have spaces before operators! Same goes for the rest of the code :)

Reply:
Fixed